cves/CVE-2021-25749.yaml

```diff
@@ -0,0 +1,19 @@
+cve: CVE-2021-25749
+issueUrl: https://github.com/kubernetes/kubernetes/issues/112192
+published: 2022-09-15T00:00Z
+description: A security issue was discovered in Kubernetes that could allow Windows workloads to run as ContainerAdministrator even when those workloads set the runAsNonRoot option to true.
+components:
+  - kubelet
+cvss:
+  kubernetes:
+    scoreV3: 7.8
+    vectorV3: CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H
+affected:
+  - range: ">= 1.20, <= 1.22.13"
+    fixedBy: "1.22.14"
+  - range: ">= 1.23.0, <= 1.23.10"
+    fixedBy: "1.23.11"
+  - range: ">= 1.24.0, <= 1.24.4"
+    fixedBy: "1.24.5"
+  - range: ">= 1.25, < 1.25.0"
+    fixedBy: "1.25.0"
```

Comment:
Is it worth mentioning 1.25.0 at all? Seems like 1.25 is not affected.

Reply:
eh. The issue explicitly calls out 1.25.0 as being a fixed version, so interpret that as meaning something like 1.25.0-alpha may be affected. I think it's ok as-is